src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -19,7 +19,8 @@
  * Represents <a href="https://www.stellar.org/developers/learn/concepts/transactions.html" target="_blank">Transaction</a> in Stellar network.
  */
 public class Transaction {
-  private static final int BASE_FEE = 100;
+  private static final int MINIMUM_BASE_FEE = 100;
+  private static int BASE_FEE = MINIMUM_BASE_FEE;
```

Comment:
It's a [convention](https://www.oracle.com/technetwork/articles/javaee/codeconventions-135099.html) to use uppercase letters for constants.

Reply:
yeah, my bad. will fix, thanks!